.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,11 @@
-#### What does this PR do?
+#### Pull request shows commits on branch classes  to be merged into branch develop
```

Comment:
You are not supposed to edit this file locally. When creating a pull request on github, you are supposed to fill in the necessary details specifying why you are making the PR.

Reply:
Got it